src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -92,8 +92,16 @@ void Snapshot::buildGraph() {
                     continue;
                 }
 
-                if (!m_graph->linked(nh, m_node_to_gr->at(ov_node_child)->getHandle())) {
-                    m_graph->link(nh, m_node_to_gr->at(ov_node_child)->getHandle());
+                Group::GPtr gr_child;
+                try {
+                    gr_child = m_node_to_gr->at(ov_node_child);
+                } catch (...) {
+                    std::cout << ov_node_child << "was not found while processing " << ov_node << std::endl;
+                    throw;
+                }
+
+                if (!m_graph->linked(nh, gr_child->getHandle())) {
+                    m_graph->link(nh, gr_child->getHandle());
```

Comment:
Left from the debug, should probably be removed

Reply:
Since (it seems) it is quite a common problem with newer versions of the models, I will solve it once and for all. :)

Reply:
replaced with `_()`